setup.py

```diff
@@ -24,9 +24,11 @@
     "jinja2",
     "ploomber-core>=0.2",
     'importlib-metadata;python_version<"3.8"',
+    "black",
```

Comment:
black is a development dependency, please move to DEV

Reply:
ah, good catch. in sk-evaluation we're using it as a runtime dependency. I'll add a little note there